src/schema/v1/suggested_genes.ts

```diff
@@ -1,4 +1,4 @@
-import fetch from "../lib/apis/fetch"
+import fetch from "lib/apis/fetch"
```

Comment:
I should probably stop pointing these out now

Reply:
I notice that VSCode auto-import will relatively import items that are one level deep but then absolutely import everything else. I wonder if that's configurable.

Reply:
It might be. That relative import path would've been two levels if corrected. Using the root src directory import notation just makes it a little more... portable?